app/services/open_graph.rb

```diff
@@ -0,0 +1,77 @@
+# This class provides methods for getting structured metadata properties for
+# Open Graph usage
+#
+# @note A wrapper around the metainspector gem
+class OpenGraph
+  delegate :meta, :meta_tags, :images, to: :page
+
+  attr_accessor :page, :tags
+
+  DEFAULT_METHODS = %i[author title description].freeze
```

Comment:
Shall we add `image` (for an optional Twitter Card image) and `favicon`?

Reply:
good call -- we most definitely should

Reply:
it may not get added to this constant, though. now that I think about it, the name of `DEFAULT_METHODS` is a little misleading. it's really a list of methods that support the `best_` prefix and unfortunately image isn't one. but, it does still have a concept of "best" image which I think still makes sense to add.